charts/prompt-proto-service/templates/prompt-proto-service.yaml

```diff
@@ -42,6 +42,12 @@ spec:
         env:
         - name: WORKER_RESTART_FREQ
           value: {{ .Values.worker.restart | toString | quote }}
+        - name: WORKER_TIMEOUT
+          value: {{ .Values.worker.timeout | toString | quote }}
+        - name: WORKER_GRACE_PERIOD
+          value: {{ .Values.worker.grace_period | toString | quote }}
+        {{- /* Knative doesn't allow timeouts longer than 1200 seconds */ -}}
```

Comment:
Timeouts can be increased by changing the config-defaults config map value of  `max-revision-timeout-seconds: "1200"` to a higher value.